### PR TITLE
Jaxb

### PR DESCRIPTION
**Pull request by lucasz-student**
- Annotations on Student and Gym model classes
- New XML file for unmarshalling
- Unmarshalling Gym object

### CODE REVIEW

**Review by jonnyfromralphs — APPROVED** [review]
Good job. I am not sure if it is the right way, but I think it would be better to make the pull request to a different branch so all the files are not in the pull request.

File: JDBC/DatabasesPractice/src/main/java/com/laba/solvd/database/DAO/GymDAO.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+			DeleteGym.setInt(1, gym.getId());
+			DeleteGym.executeUpdate();
+		} catch (SQLException | InterruptedException | ExecutionException e) {
+			e.printStackTrace();
+		}
+	}
+	
+	public void updateGym(Gym gym) {
+		try (Connection connection = (Connection) ConnectionManager.getSQLConnection().get();) {
+			PreparedStatement UpdateGym = connection.prepareStatement("UPDATE gym set Location=?, set gymAge=?, set price=? WHERE id=?");
```
Comment (zhyldyzMaks) [inline JDBC/DatabasesPractice/src/main/java/com/laba/solvd/database/DAO/GymDAO.java:89]: I think no need to repeat set keyword.

File: JDBC/DatabasesPractice/src/main/java/com/laba/solvd/database/DAO/GymDAO.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+public class GymDAO {
+
+	public void createGym(Gym gym) {
+		try (Connection connection = (Connection) ConnectionManager.getSQLConnection().get();) {
```
Comment (zhyldyzMaks) [inline JDBC/DatabasesPractice/src/main/java/com/laba/solvd/database/DAO/GymDAO.java:14]: What is the purpose of casting Connection object here? I just wonder if it works without casting.

**Review by zhyldyzMaks — APPROVED** [review]
I marked a couple of minor points. The rest looks good!

**Pull request merged**